core/src/main/scala/org/apache/spark/deploy/master/MasterMessages.scala

```diff
@@ -36,7 +36,7 @@ private[master] object MasterMessages {
 
   case object CompleteRecovery
 
-  case object RequestWebUIPort
```

Comment:
Is this just changed for clarity? If so, maybe we should avoid changing it since it will break older akka clients.